tools/service.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# This script can be used by cron to auto restart Datalad-Registry service upon
+# the host's reboot.
+
+set -eu
+umask 077
+
+# Check if the path to the Datalad-Registry repo is provided
+if [ -z "${1:-}" ]; then
+    echo "Usage: $0 <directory-path of Datalad-Registry repo>"
+    exit 1
+fi
+
+cd "$1"
```

Comment:
overall -- are you starting from this repo? if so -- why didn't you take the pattern I gave you in https://github.com/datalad/datalad-registry/issues/299 ie 

```
cd $(dirname $0) && cd ..
```
and thus avoiding needing `$1` altogether?

Reply:
No, I am not starting from this repo. I am starting from https://github.com/candleindark/datalad-registry, and depending on what latest features we want to include (also tests to an extend),  I start from a branch other than the master branch sometime.

Reply:
Wait, by repo. Do you mean the containing folder? 

I thought that cron's current working directory may not be the directory containing the script.

However, if cron's current working directory is not guarantee to be the directory containing the script, I thing I have to change the following line as well.

`podman-compose -f docker-compose.dev.yml --env-file .env.dev.typhon up -d --build`

Is cron guaranteed to have a current working directory that is the directory the script resides?

Reply:
> I thought that cron's current working directory may not be the directory containing the script.

correct.

> Is cron guaranteed to have a current working directory that is the directory the script resides?

not guaranteed.

That is why that `cd $(dirname $0)` I recommended -- to get to the directory containing the script! This way you do not need to provide long path to the script and then again that script as argument.

this all has nothing to do with branches.

Reply:
The script has been updated, and the `cd $(dirname $0) && cd ..` pattern is now used in the script.